packages/react-components/src/components/Tag/Tag.tsx

```diff
@@ -25,12 +25,27 @@ export interface TagProps extends ReactAriaTagProps {
     | "green"
     | "red"
     | "yellow";
+  /**
+   * Style
+   */
+  style?: "rectangular" | "circular";
```

Comment:
The `style` prop name [already exists](https://react-spectrum.adobe.com/react-aria/TagGroup.html#tag) on RAC Tag, providing a method for injecting inline CSS into a component. 

I'd suggest we rename this to `variant` (following the pattern of our other components, like Button.)